man/adoc/bpftrace.adoc

```diff
@@ -1287,6 +1287,25 @@ t:syscalls:sys_enter_execve {
 55f683ee2000-55f683ee3000 rw-p 00000000 00:00 0
 ----
 
+=== cgroup_path
+
+.variants
+* `cgroup_path cgroup_path(int cgroupid)`
+
+Convert cgroup id to cgroup path.
+This is done asynchronously in userspace when the cgroup_path value is printed,
+therefore it can resolve to a different value if the cgroup id gets reassigned.
```

Comment:
Just curious, how likely is this?

Reply:
Very unlikely, the reassignment would have to happen exactly between the two perf event handler calls, perhaps not possible at all if cgroup ids don't get reused.